sysl2/sysl/fmt_grammar/FmtParser.g4

```diff
@@ -0,0 +1,60 @@
+grammar FmtParser;
+
+expression
+    : text* ((simp_exp
+                 | bool_exp
+                 | cmp_exp
+                 | text) text*)*
+    ;
+yes_stmt
+    : text* ((simp_exp
+                 | bool_exp
+                 | cmp_exp
+                 | text) text*)*
+    ;
+no_stmt
+    : text* ((simp_exp
+                 | bool_exp
+                 | cmp_exp
+                 | text) text*)*
+    ;
```

Comment:
what is the difference between `expression`, `yes_stmt` and `no_stmt`?

Reply:
no difference, but convenience for me to do the implement.
like `a==b?yes|no`, it's convenience for me to pick wether yes or no.

Reply:
i suggest you use expression for yes_stmt and same for no_stmt.